forge-gui/res/cardsfolder/g/grothama_all_devouring.txt

```diff
@@ -10,4 +10,6 @@ SVar:TrigRepeat:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ TrigD
 SVar:TrigDraw:DB$ Draw | Defined$ Remembered | NumCards$ X
 SVar:X:TriggeredCard$TotalDamageThisTurn Card.RememberedPlayerCtrl
 SVar:HasAttackEffect:TRUE
+#NOTE: AI will always choose to activate the "fight Grothama" ability if one of its creatures attacks.
+AI:RemoveDeck:All
```

Comment:
This is a pretty universal issue, not only for the AI deck - if the human plays one from his own deck, it'll also be very abuse-worthy. I'll think about how to improve the AI handling of this card so that it doesn't work as badly as it does right now :)